From 586aaf5e0be365718e9b7c93c7201f4e89c384d5 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 18 Jul 2017 15:19:16 +0800
Subject: [PATCH 2154/5242] MLK-16015-5 drm/imx: ldb: Specify pixel link
 quirks to be initialization related

commit  4625bde6fccc0d32e84688c639790d3714fe61cb from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch specifies the existing pixel link quirks is initialization related.
This may help us distinguish between the pixel link quirks and another one
up-coming which is validation and invalidation related.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/imx-ldb.c |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/gpu/drm/imx/imx-ldb.c b/drivers/gpu/drm/imx/imx-ldb.c
index 7b9766e..a9e23b6 100644
--- a/drivers/gpu/drm/imx/imx-ldb.c
+++ b/drivers/gpu/drm/imx/imx-ldb.c
@@ -114,7 +114,7 @@ struct devtype {
 	bool use_mixel_phy;
 	bool use_mixel_combo_phy;
 	bool padding_quirks;
-	bool pixel_link_quirks;
+	bool pixel_link_init_quirks;
 
 	/* pixel rate in KHz */
 	unsigned int max_prate_single_mode;
@@ -142,7 +142,7 @@ struct imx_ldb {
 	bool use_mixel_phy;
 	bool use_mixel_combo_phy;
 	bool padding_quirks;
-	bool pixel_link_quirks;
+	bool pixel_link_init_quirks;
 
 	/* pixel rate in KHz */
 	unsigned int max_prate_single_mode;
@@ -826,7 +826,7 @@ static u32 of_get_bus_format(struct device *dev, struct imx_ldb *ldb,
 	.is_imx8 = true,
 	.use_mixel_combo_phy = true,
 	.padding_quirks = true,
-	.pixel_link_quirks = true,
+	.pixel_link_init_quirks = true,
 	.max_prate_single_mode = 150000,
 	.max_prate_dual_mode = 300000,
 };
@@ -888,9 +888,9 @@ static int imx_ldb_panel_ddc(struct device *dev,
 }
 
 #ifndef CONFIG_HAVE_IMX8_SOC
-static void ldb_pixel_link_config(int id) {}
+static void ldb_pixel_link_init(int id) {}
 #else
-static void ldb_pixel_link_config(int id)
+static void ldb_pixel_link_init(int id)
 {
 	sc_err_t sciErr;
 	sc_ipc_t ipcHndl = 0;
@@ -968,7 +968,7 @@ static int imx_ldb_bind(struct device *dev, struct device *master, void *data)
 	imx_ldb->use_mixel_phy = devtype->use_mixel_phy;
 	imx_ldb->use_mixel_combo_phy = devtype->use_mixel_combo_phy;
 	imx_ldb->padding_quirks = devtype->padding_quirks;
-	imx_ldb->pixel_link_quirks = devtype->pixel_link_quirks;
+	imx_ldb->pixel_link_init_quirks = devtype->pixel_link_init_quirks;
 	imx_ldb->max_prate_single_mode = devtype->max_prate_single_mode;
 	imx_ldb->max_prate_dual_mode = devtype->max_prate_dual_mode;
 
@@ -1121,9 +1121,9 @@ static int imx_ldb_bind(struct device *dev, struct device *master, void *data)
 
 	dev_set_drvdata(dev, imx_ldb);
 
-	if (imx_ldb->pixel_link_quirks) {
+	if (imx_ldb->pixel_link_init_quirks) {
 		imx_ldb->id = of_alias_get_id(np, "ldb");
-		ldb_pixel_link_config(imx_ldb->id);
+		ldb_pixel_link_init(imx_ldb->id);
 	}
 
 	return 0;
-- 
1.7.9.5

